rank=38 L00: Traceback (most recent call last):
rank=38 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=38 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=38 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=38 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=38 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=38 L06:     f(args)
rank=38 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=38 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=38 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=38 L10:     return _run_module_code(code, init_globals, run_name,
rank=38 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=38 L12:     _run_code(code, mod_globals, init_globals,
rank=38 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=38 L14:     exec(code, run_globals)
rank=38 L15:   File "get_me_coupling.py", line 92, in <module>
rank=38 L16:     structure.get_potential_energy()
rank=38 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=38 L18:     energy = self._calc.get_potential_energy(self)
rank=38 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=38 L20:     return self.get_property(name, atoms)
rank=38 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=38 L22:     self.calculate(atoms, [name], system_changes)
rank=38 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=38 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=38 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=38 L26:     self.set_positions(atoms)
rank=38 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=38 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=38 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=38 L30:     hamiltonian.update(density)
rank=38 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=38 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=38 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=38 L34:     dH_sp[1:] = pack2(dH_vii)
rank=38 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 38): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=39 L00: Traceback (most recent call last):
rank=39 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=39 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=39 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=39 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=39 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=39 L06:     f(args)
rank=39 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=39 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=39 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=39 L10:     return _run_module_code(code, init_globals, run_name,
rank=39 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=39 L12:     _run_code(code, mod_globals, init_globals,
rank=39 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=39 L14:     exec(code, run_globals)
rank=39 L15:   File "get_me_coupling.py", line 92, in <module>
rank=39 L16:     structure.get_potential_energy()
rank=39 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=39 L18:     energy = self._calc.get_potential_energy(self)
rank=39 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=39 L20:     return self.get_property(name, atoms)
rank=39 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=39 L22:     self.calculate(atoms, [name], system_changes)
rank=39 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=39 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=39 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=39 L26:     self.set_positions(atoms)
rank=39 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=39 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=39 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=39 L30:     hamiltonian.update(density)
rank=39 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=39 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=39 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=39 L34:     dH_sp[1:] = pack2(dH_vii)
rank=39 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 39): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=40 L00: Traceback (most recent call last):
rank=40 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=40 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=40 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=40 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=40 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=40 L06:     f(args)
rank=40 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=40 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=40 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=40 L10:     return _run_module_code(code, init_globals, run_name,
rank=40 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=40 L12:     _run_code(code, mod_globals, init_globals,
rank=40 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=40 L14:     exec(code, run_globals)
rank=40 L15:   File "get_me_coupling.py", line 92, in <module>
rank=40 L16:     structure.get_potential_energy()
rank=40 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=40 L18:     energy = self._calc.get_potential_energy(self)
rank=40 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=40 L20:     return self.get_property(name, atoms)
rank=40 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=40 L22:     self.calculate(atoms, [name], system_changes)
rank=40 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=40 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=40 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=40 L26:     self.set_positions(atoms)
rank=40 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=40 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=40 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=40 L30:     hamiltonian.update(density)
rank=40 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=40 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=40 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=40 L34:     dH_sp[1:] = pack2(dH_vii)
rank=40 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 40): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=41 L00: Traceback (most recent call last):
rank=41 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=41 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=41 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=41 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=41 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=41 L06:     f(args)
rank=41 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=41 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=41 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=41 L10:     return _run_module_code(code, init_globals, run_name,
rank=41 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=41 L12:     _run_code(code, mod_globals, init_globals,
rank=41 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=41 L14:     exec(code, run_globals)
rank=41 L15:   File "get_me_coupling.py", line 92, in <module>
rank=41 L16:     structure.get_potential_energy()
rank=41 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=41 L18:     energy = self._calc.get_potential_energy(self)
rank=41 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=41 L20:     return self.get_property(name, atoms)
rank=41 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=41 L22:     self.calculate(atoms, [name], system_changes)
rank=41 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=41 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=41 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=41 L26:     self.set_positions(atoms)
rank=41 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=41 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=41 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=41 L30:     hamiltonian.update(density)
rank=41 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=41 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=41 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=41 L34:     dH_sp[1:] = pack2(dH_vii)
rank=41 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 41): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=42 L00: Traceback (most recent call last):
rank=42 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=42 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=42 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=42 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=42 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=42 L06:     f(args)
rank=42 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=42 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=42 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=42 L10:     return _run_module_code(code, init_globals, run_name,
rank=42 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=42 L12:     _run_code(code, mod_globals, init_globals,
rank=42 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=42 L14:     exec(code, run_globals)
rank=42 L15:   File "get_me_coupling.py", line 92, in <module>
rank=42 L16:     structure.get_potential_energy()
rank=42 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=42 L18:     energy = self._calc.get_potential_energy(self)
rank=42 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=42 L20:     return self.get_property(name, atoms)
rank=42 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=42 L22:     self.calculate(atoms, [name], system_changes)
rank=42 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=42 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=42 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=42 L26:     self.set_positions(atoms)
rank=42 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=42 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=42 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=42 L30:     hamiltonian.update(density)
rank=42 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=42 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=42 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=42 L34:     dH_sp[1:] = pack2(dH_vii)
rank=42 L35: ValueError: could not broadcast input array from shape (3,171) into shape (1,171)
GPAW CLEANUP (node 42): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=43 L00: Traceback (most recent call last):
rank=43 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=43 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=43 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=43 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=43 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=43 L06:     f(args)
rank=43 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=43 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=43 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=43 L10:     return _run_module_code(code, init_globals, run_name,
rank=43 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=43 L12:     _run_code(code, mod_globals, init_globals,
rank=43 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=43 L14:     exec(code, run_globals)
rank=43 L15:   File "get_me_coupling.py", line 92, in <module>
rank=43 L16:     structure.get_potential_energy()
rank=43 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=43 L18:     energy = self._calc.get_potential_energy(self)
rank=43 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=43 L20:     return self.get_property(name, atoms)
rank=43 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=43 L22:     self.calculate(atoms, [name], system_changes)
rank=43 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=43 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=43 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=43 L26:     self.set_positions(atoms)
rank=43 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=43 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=43 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=43 L30:     hamiltonian.update(density)
rank=43 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=43 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=43 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=43 L34:     dH_sp[1:] = pack2(dH_vii)
rank=43 L35: ValueError: could not broadcast input array from shape (3,171) into shape (1,171)
rank=44 L00: Traceback (most recent call last):
rank=44 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=44 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=44 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=44 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=44 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=44 L06:     f(args)
rank=44 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=44 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=44 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=44 L10:     return _run_module_code(code, init_globals, run_name,
rank=44 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=44 L12:     _run_code(code, mod_globals, init_globals,
rank=44 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=44 L14:     exec(code, run_globals)
rank=44 L15:   File "get_me_coupling.py", line 92, in <module>
rank=44 L16:     structure.get_potential_energy()
rank=44 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=44 L18:     energy = self._calc.get_potential_energy(self)
rank=44 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=44 L20:     return self.get_property(name, atoms)
rank=44 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=44 L22:     self.calculate(atoms, [name], system_changes)
rank=44 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=44 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=44 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=44 L26:     self.set_positions(atoms)
rank=44 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=44 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=44 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=44 L30:     hamiltonian.update(density)
rank=44 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=44 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=44 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=44 L34:     dH_sp[1:] = pack2(dH_vii)
rank=44 L35: ValueError: could not broadcast input array from shape (3,171) into shape (1,171)
GPAW CLEANUP (node 44): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=45 L00: Traceback (most recent call last):
rank=45 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=45 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=45 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=45 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=45 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=45 L06:     f(args)
rank=45 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=45 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=45 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=45 L10:     return _run_module_code(code, init_globals, run_name,
rank=45 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=45 L12:     _run_code(code, mod_globals, init_globals,
rank=45 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=45 L14:     exec(code, run_globals)
rank=45 L15:   File "get_me_coupling.py", line 92, in <module>
rank=45 L16:     structure.get_potential_energy()
rank=45 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=45 L18:     energy = self._calc.get_potential_energy(self)
rank=45 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=45 L20:     return self.get_property(name, atoms)
rank=45 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=45 L22:     self.calculate(atoms, [name], system_changes)
rank=45 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=45 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=45 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=45 L26:     self.set_positions(atoms)
rank=45 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=45 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=45 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=45 L30:     hamiltonian.update(density)
rank=45 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=45 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=45 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=45 L34:     dH_sp[1:] = pack2(dH_vii)
rank=45 L35: ValueError: could not broadcast input array from shape (3,171) into shape (1,171)
GPAW CLEANUP (node 45): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=46 L00: Traceback (most recent call last):
rank=46 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=46 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=46 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=46 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=46 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=46 L06:     f(args)
rank=46 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=46 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=46 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=46 L10:     return _run_module_code(code, init_globals, run_name,
rank=46 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=46 L12:     _run_code(code, mod_globals, init_globals,
rank=46 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=46 L14:     exec(code, run_globals)
rank=46 L15:   File "get_me_coupling.py", line 92, in <module>
rank=46 L16:     structure.get_potential_energy()
rank=46 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=46 L18:     energy = self._calc.get_potential_energy(self)
rank=46 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=46 L20:     return self.get_property(name, atoms)
rank=46 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=46 L22:     self.calculate(atoms, [name], system_changes)
rank=46 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=46 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=46 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=46 L26:     self.set_positions(atoms)
rank=46 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=46 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=46 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=46 L30:     hamiltonian.update(density)
rank=46 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=46 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=46 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=46 L34:     dH_sp[1:] = pack2(dH_vii)
rank=46 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 46): <class 'ValueError'> occurred.  Calling MPI_Abort!
rank=47 L00: Traceback (most recent call last):
rank=47 L01:   File "/home/niflheim/madkru/new_venv/bin/gpaw", line 33, in <module>
rank=47 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=47 L03:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=47 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=47 L05:   File "/home/niflheim/madkru/new_venv/ase/ase/cli/main.py", line 103, in main
rank=47 L06:     f(args)
rank=47 L07:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=47 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=47 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=47 L10:     return _run_module_code(code, init_globals, run_name,
rank=47 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=47 L12:     _run_code(code, mod_globals, init_globals,
rank=47 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=47 L14:     exec(code, run_globals)
rank=47 L15:   File "get_me_coupling.py", line 92, in <module>
rank=47 L16:     structure.get_potential_energy()
rank=47 L17:   File "/home/niflheim/madkru/new_venv/ase/ase/atoms.py", line 731, in get_potential_energy
rank=47 L18:     energy = self._calc.get_potential_energy(self)
rank=47 L19:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/abc.py", line 24, in get_potential_energy
rank=47 L20:     return self.get_property(name, atoms)
rank=47 L21:   File "/home/niflheim/madkru/new_venv/ase/ase/calculators/calculator.py", line 509, in get_property
rank=47 L22:     self.calculate(atoms, [name], system_changes)
rank=47 L23:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 331, in calculate
rank=47 L24:     for _ in self.icalculate(atoms, properties, system_changes):
rank=47 L25:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 354, in icalculate
rank=47 L26:     self.set_positions(atoms)
rank=47 L27:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/calculator.py", line 554, in set_positions
rank=47 L28:     nlcao, nrand = self.wfs.initialize(self.density, self.hamiltonian,
rank=47 L29:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/wavefunctions/fdpw.py", line 307, in initialize
rank=47 L30:     hamiltonian.update(density)
rank=47 L31:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 232, in update
rank=47 L32:     atomic_energies = self.update_corrections(density, W_aL)
rank=47 L33:   File "/home/niflheim/madkru/new_venv/gpaw/gpaw/hamiltonian.py", line 284, in update_corrections
rank=47 L34:     dH_sp[1:] = pack2(dH_vii)
rank=47 L35: ValueError: could not broadcast input array from shape (3,91) into shape (1,91)
GPAW CLEANUP (node 47): <class 'ValueError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 43): <class 'ValueError'> occurred.  Calling MPI_Abort!
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 40 in communicator MPI_COMM_WORLD
with errorcode 42.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
[x056.nifl.fysik.dtu.dk:01052] 9 more processes have sent help message help-mpi-api.txt / mpi-abort
[x056.nifl.fysik.dtu.dk:01052] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
